<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task2Window</class>
 <widget class="QMainWindow" name="Task2Window">
  <property name="geometry"><rect><x>0</x><y>0</y><width>600</width><height>400</height></rect></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item><widget class="QLineEdit" name="le_event_name"><property name="placeholderText"><string>Название события</string></property></widget></item>
    <item><widget class="QCalendarWidget" name="calendar"/></item>
    <item><widget class="QTimeEdit" name="timeEdit"/></item>
    <item><widget class="QPushButton" name="btn_add_event"><property name="text"><string>Добавить</string></property></widget></item>
    <item><widget class="QListWidget" name="listEvents"/></item>
   </layout>
  </widget>
 </widget>
 <resources/><connections/>
</ui>